api/client/rm.go

```diff
@@ -30,6 +30,7 @@ func (cli *DockerCli) CmdRm(args ...string) error {
 
 	var encounteredError error
 	for _, name := range cmd.Args() {
+		name = stripPrefix(name)
```

Comment:
Should we use [`strings.TrimPrefix`](http://golang.org/pkg/strings/#TrimPrefix) here?